MonMaraicher/Model/Farmer/Farmers.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  Produceurs.swift
+//  MonMaraicher
+//
+//  Created by Yves Charpentier on 10/04/2024.
+//
+
+import Foundation
+
+struct Farmers: Decodable, Hashable {
+    let nbTotal: String
+    let items: [Farmer]
+}
+
+struct Farmer: Decodable, Identifiable, Hashable {
+    let id: Int
+    let raisonSociale: String
```

Comment:
You need to fix all french word

Reply:
I would have liked it because it’s simply the naming of the JSON that I kept

Reply:
So if the API is returning naming like `udneaounda09jd3` you will keep it like it and use it in your project? 😅

Reply:
Absolutely not! I have always learned that it was necessary to strictly take each value to do parsing and not make a mistake, and so I always did it like that and I didn’t know that we could do otherwise.